### DISCUSSION TITLE
[Run cases/plans] Display only project environments instead of all projects environments

### DISCUSSION THREAD

**Discussion by PavelMal** [allure-framework/allure2#2086] [TestOps]
Hi there!

In case of running cases/plans it would be useful to display only project environments. 
For now Allure TestOps displays every environments (even from other projects). 
Maybe we can resolve it by adding checkbox such as: `use only project envs`

<img width="1358" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/9362f021-dc24-4e2f-a7a4-cb0f0dce724a">

**Comment by cheshi-mantu** [comment]
@PavelMal Hi sir,
this is being fixed as we speak and will be available in coming release.
Please consider sending your suggestions and bug fixes via dedicated support portal https://help.qameta.io
